charts/drax/charts/network-state-monitor/Chart.yaml

```diff
@@ -2,9 +2,9 @@ apiVersion: v2
 name: network-state-monitor
 description: The Accelleran dRAX Network State Monitor
 type: application
-version: 0.1.0
+version: 0.2.0
```

Comment:
Can be kept to `0.1.1` as it's a feature update pre 1.0.0. So we bump minor only for breaking changes and patch for features and bug fixes. Note that the CI fails because this version will now not match anymore with the one in `charts/drax/Chart.yaml` and `charts/drax/Chart.lock`. The last file should be changed with `helm dependency update charts/drax`.

Reply:
ok will keep it to 0.1.1